fhir-server-test/src/test/java/com/ibm/fhir/server/test/ExportOperationTest.java

```diff
@@ -280,26 +293,30 @@ public void testGroup() throws Exception {
         savedGroupId2 = getLocationLogicalId(response);
 
         // Next, call the 'read' API to retrieve the new group and verify it.
-        response = target.path("Group/" + savedGroupId2).request(FHIRMediaType.APPLICATION_FHIR_JSON).get();
+        response      = target.path("Group/" + savedGroupId2).request(FHIRMediaType.APPLICATION_FHIR_JSON).get();
```

Comment:
space

Reply:
fixed, I was playing with save and alignment